metron-interface/metron-alerts/e2e/utils/clean_metron_update_table.ts

```diff
@@ -0,0 +1,52 @@
+declare var Promise: any;
+var chalk = require('chalk');
+var Client = require('ssh2').Client;
+var errorMsg = '';
+
+export function cleanMetronUpdateTable() {
+  return  new Promise(
+      function (resolve, reject) {
+        // resolve();
+        cleanupTable(resolve, reject);
```

Comment:
IMHO, it would be better to default to `resolve()` only owing to the issues with Vagrant `ssh2` client.